### PR TITLE
play notification sound if message has keywords, bump version to 0.6.1, only poll enabled domains

### PR DESCRIPTION
**Pull request by HappyDOGE**

### CODE REVIEW

**Comment by april83c** [comment]
did you test this? from our testing we can't load audio from outside reddit.com

**Comment by HappyDOGE** [comment]
this does not 'load' audio through official means (HTMLAudioElement), but instead uses the GM_xhr API to bypass CSP and get the audio data as ArrayBuffer, which is then treated as audio data without accompanying source. This data can be played using AudioContext, which does not care about the source of the data.
And yes it works in both Chrome and Firefox.

**Comment by HappyDOGE** [comment]
*worth noting is that it will only play is the user has interacted on the page beforehand, just as regular Audio element does

**Comment by april83c** [comment]
@HappyDOGE could you make it so that all messages play the sound?

**Comment by HappyDOGE** [comment]
@april83c done

**Comment by april83c** [comment]
i'm not merging this yet because of the bug where when it's open for a while it has multiple broadcaster connections (i'm not sure if it's fixed yet) but i'll merge it if we fix that

File: src/ui/notificationsManager.ts [hunk 1]
```diff
@@ -1,11 +1,44 @@
 import * as utils from "../utils";
 
+const NOTIFICATION_SOUND_KEYWORDS = ['!important!'];
```
Comment (april83c) [inline src/ui/notificationsManager.ts:3]: this is redundant

File: src/ui/notificationsManager.ts [hunk 2]
```diff
@@ -1,11 +1,44 @@
...
+const NOTIFICATION_SOUND_URL = 'https://files.catbox.moe/c9nwlu.mp3';
```
Comment (april83c) [inline src/ui/notificationsManager.ts:4]: i'm not sure how i feel about this being hardcoded. maybe it's fine for now but in the future i would definitely want to make this an option

File: src/ui/notificationsManager.ts [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
+                } catch (e) {
+                    error(e);
+                }
+            },
+            onerror: error
+        });
+    }
+
+    messageNeedsSound(message: string): boolean {
+        return true; // NOTIFICATION_SOUND_KEYWORDS.some((kw) => message.includes(kw));
```
Comment (april83c) [inline src/ui/notificationsManager.ts:41]: this function is redundant

**Review by april83c — COMMENTED** [review]
LGTM other than these small things

**Review by april83c — APPROVED** [review]
LGTM, tested and seems to work

**Pull request merged**